stock_reserve/models/stock_reserve.py

```diff
@@ -122,8 +122,10 @@ def reserve(self):
         A date until which the product is reserved can be specified.
         """
         self.write({"date_deadline": fields.Datetime.now()})
-        self.mapped("move_id")._action_confirm(merge=False)
-        self.mapped("move_id.picking_id").action_assign()
+        self.mapped("move_id").with_context(skip_assign_picking=True)._action_confirm(
```

Comment:
I think this is simpler: just don't call `_action_confirm`. It's not doing anything we want if I don't oversee anything.